yellowbrick/__init__.py

```diff
@@ -31,6 +31,7 @@
 
 # Import yellowbrick functionality to the top level
 from .anscombe import anscombe
+from .classifier import ROCAUC, ClassBalance, ClassificationScoreVisualizer
```

Comment:
Ok, I think I see what you mean now; @rebeccabilbro and I were debating whether or not to hoist everything into the top level like NLTK does or to keep things in their packages. To import these tools you'd do:

```python
from yellowbrick.classifier import ROCAUC, ClassBalance, ClassificationScoreVisualizer 
```

My guess is that you're doing something similar to:

```python
import yellowbrick as yb 

yb.ClassBalance()
```

Is that right? 

Frankly, I'm torn about whether or not to hoist these to the top level; I'd say things need to be hoisted to the second level (e.g. text, features, classifier, or regressor). Since we're going to be creating directory models for these soon, my sense is that we should leave them as is.